github/ci/prow-deploy/files/jobs/kubevirt/application-aware-quota/application-aware-quota-postsubmits.yaml

```diff
@@ -165,25 +166,25 @@ postsubmits:
     decoration_config:
       timeout: 3h
       grace_period: 5m
+    max_concurrency: 1
     labels:
       preset-podman-in-container-enabled: "true"
       preset-docker-mirror-proxy: "true"
       preset-gcs-credentials: "true"
       preset-github-credentials: "true"
       preset-kubevirtci-quay-credential: "true"
     spec:
+      nodeSelector:
+        type: bare-metal-external
```

Comment:
this node selector is preventing the job from being scheduled as there are no nodes with this label in the cluster that this job is targeting.

Reply:
Ah, i copied that from the `publish-kubevirt-builder` job, however that one runs on the `prow-workloads` cluster, while this one is targeting `kubevirt-prow-control-plane`.
What is the correct choice here, since we need kvm emulation for building a multiarch manifest?

Reply:
Ahh yes it should go to `prow-workloads` if it requires emulation - the performance would be too poor on `kubevirt-prow-control-plane`.

Reply:
I have raised https://github.com/kubevirt/project-infra/pull/4148 as a follow up